cli/cmd/plugin/ui/web/tanzu-ui/src/views/unmanaged-cluster/UnmanagedClusterWizard.tsx

```diff
@@ -0,0 +1,53 @@
+// React imports
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+
+// Library imports
+import { CdsIconButton, CdsButton } from '@cds/react/button';
+import { CdsIcon } from '@cds/react/icon';
+import { arrowIcon, ClarityIcons } from '@cds/core/icon';
+
+// App imports
+import { UmcProvider } from '../../state-management/stores/Store.umc';
+import UnmanagedClusterWizardBasic from './unmanaged-cluster-wizard-page/UnmanagedClusterWizardBasic';
+import UnmanagedClusterWizardAdvanced from './unmanaged-cluster-wizard-page/UnmanagedClusterWizardAdvanced';
+import { NavRoutes } from '../../shared/constants/NavRoutes.constants';
+
+ClarityIcons.addIcons(arrowIcon);
+
+function UnmanagedClusterWizard() {
+    const navigate = useNavigate();
+    // temp local variable for testing advanced settings, will be refactored
+    const setting_switch = true;
```

Comment:
I know this is just a temporary variable (thanks for the comment), but I'd still suggest naming it "nicely", so the name tells you what it's used for. Something like `useBasicSettings`.